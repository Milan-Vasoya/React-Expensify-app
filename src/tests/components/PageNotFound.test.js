import React from 'react';
import {shallow} from 'enzyme' ;
import PageNotFound from '../../components/PageNotFound';

test('Should add three item in expense list ' ,()=>{
    const wrapper = new shallow(<PageNotFound />);

    expect(wrapper).toMatchSnapshot();
});