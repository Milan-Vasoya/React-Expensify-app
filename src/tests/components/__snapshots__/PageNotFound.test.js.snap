// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should add three item in expense list  1`] = `
<div>
  404 
  <Link
    to="/expensify"
  >
    Go To Home
  </Link>
</div>
`;
